dev-tools/reconfigurator-cli/tests/output/cmds-example-stdout

```diff
@@ -1013,6 +1013,411 @@ parent:    02697f74-b14a-4418-90f0-c28b2a3a6aa9
+        ec93753f-fcd0-4624-8dcc-328dabb3f322         oxp_ab94a812-86ce-428c-bbbb-6ce1ab0b071b/crypt/zone/oxz_nexus_d856156c-2bc2-41ad-beef-7ca1da5802d3                   off                 none            none        
+        ff1c7f38-0cf7-4d8a-bc78-b0dd66d22d72         oxp_0477165a-a72e-4814-b8d6-74aa02cb2040/crypt/zone/oxz_clickhouse_a27817ff-db8d-49ba-9682-9e3c13bc4e4b              off                 none            none        
+    ZONES: 17
+        ID                                           KIND                    IMAGE_SOURCE    
+        03ba2f16-8b5e-41c4-906d-5a7a0af460ef         external_dns            install-dataset 
+        056fd45c-2498-4ce4-8c97-79dfc9321080         nexus                   install-dataset 
+        17ed8f62-8cd0-416c-86d1-5eaf6f7358ef         crucible_pantry         install-dataset 
+        397841de-588e-4fe7-94ee-a944b7340074         crucible                install-dataset 
+        4692cc31-6eb6-437c-9634-9688663d06ae         internal_dns            install-dataset 
+        4a81477d-5a28-4246-93ef-01db835a5c5e         crucible                install-dataset 
+        534dad87-2b4c-4250-8ad9-2ec1d93b2994         crucible_pantry         install-dataset 
+        587da9e8-8fc0-4854-b585-070741a7b00d         internal_dns            install-dataset 
+        5954ecdc-c5ad-49a6-9c5f-4689f783ba83         external_dns            install-dataset 
+        83ac30bd-ae85-4d6b-84c3-add02d3214cf         external_dns            install-dataset 
+        a27817ff-db8d-49ba-9682-9e3c13bc4e4b         clickhouse              install-dataset 
+        a67ac9b3-427b-4ea6-a891-1c76a22720f5         nexus                   install-dataset 
+        ac5bb28e-91d5-42f3-a57a-d84e1c414c17         internal_ntp            install-dataset 
+        d856156c-2bc2-41ad-beef-7ca1da5802d3         nexus                   install-dataset 
+        dc2666e6-4c3e-4b8e-99bc-bcdb5f8986e1         crucible_pantry         install-dataset 
+        f4dc5b5d-6eb6-40a9-a079-971eca862285         crucible                install-dataset 
+        ffbf02f0-261d-4723-b613-eb861245acbd         internal_dns            install-dataset 
+    boot disk slot: FAILED TO DETERMINE: constructed via debug_assume_success()
+    slot A details UNAVAILABLE: constructed via debug_assume_success()
+    slot B details UNAVAILABLE: constructed via debug_assume_success()
+    last reconciled config: matches ledgered config
+        no orphaned datasets
+        all disks reconciled successfully
+        all datasets reconciled successfully
+        all zones reconciled successfully
+    reconciler task status: idle (finished at <REDACTED_TIMESTAMP> after running for <REDACTED_DURATION>s)
+ORPHANED DATASETS
+
+sled 2eb69596-f081-4e2d-9425-9994926e0832 (serial serial1)
+    no orphaned datasets
+
+sled 32d8d836-4d8a-4e54-8fa9-f31d79c42646 (serial serial2)
+    no orphaned datasets
+
+sled 89d02b1b-478c-401a-8e28-7a26f74fa41b (serial serial0)
+    no orphaned datasets
```

Comment:
(This comment is about #8613 but is easier to show here.)

This is the section I think we _don't_ want by default, since this is already reported inside the ledgered config results of each sled above. But we want to be able to ask for this section on its own, so we have a quick way to check for orphans.

Reply:
Addressed in #8613 -- thanks!